library/tasks/CreateCountMatrix.wdl

```diff
@@ -118,24 +119,12 @@ task CreateSparseCountMatrix {
 }
 
 task MergeCountFiles {
-<<<<<<< HEAD
-  Array[File] sparse_count_matrices
-  Array[File] row_indices
-  Array[File] col_indices
-  String counting_mode = "sc_rna"
-
-  # runtime values
-  String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v0.3.2"
-  Int machine_mem_mb = 8250
-  Int cpu = 1
-  Int disk = 20  # todo find out how to make this adaptive with Array[file] input
-  Int preemptible = 3
-=======
   input {
     Array[File] sparse_count_matrices
     Array[File] row_indices
     Array[File] col_indices
-
+    String counting_mode = "sc_rna"
```

Comment:
Is this input used in this task?

Reply:
It will be when we finish the pipeline -- so required. The reason these are commented out is that our counting step doesn't support this yet.

Reply:
👍